fem/bilinearform_ext.cpp

```diff
@@ -955,6 +955,10 @@ void FABilinearFormExtension::Assemble()
       }
       a->mat = mat;
    }
+   if ( a->sort_device_matrix && Device::Allows( Backend::DEVICE_MASK ) )
```

Comment:
Do we want `Device::Allows(Backend::DEVICE_MASK) ` or `Device::Allows(~Backend::CPU_MASK)`? Maybe we want this turned on with OpenMP, which isn't included in `DEVICE_MASK`. cc: @v-dobrev

Reply:
I guess it's on the safer side. [7404fbb](https://github.com/mfem/mfem/pull/3172/commits/7404fbbaf112ec8b7f58628b713c16980dd8f875)